microservices.yml

```diff
@@ -263,14 +263,133 @@ commands:
             "prod")
                 echo "Using prod resources"
                 echo "export SVC_CONFIG_S3_BUCKET=mgmri-mobapp-production/services-configurations" >> $BASH_ENV
+                echo 'export TERRAFORM_STATE_S3_BUCKET=mgmri-services-tf-state' >> $BASH_ENV
                 ;;
             *)
                 echo "Using non-prod resources"
                 echo "export SVC_CONFIG_S3_BUCKET=mgmresorts-services-configurations" >> $BASH_ENV
+                echo 'export TERRAFORM_STATE_S3_BUCKET=mgmresorts-services-tf-state' >> $BASH_ENV
                 ;;
             esac
 
+  install-terraform-and-npm:
+    description: Install terraform and update npm (for npm ci)
+    steps:
+      - run:
+          name: Install terraform and update npm
+          command: |
+            npm i -g npm@latest
```

Comment:
@patrick-sf I'm now realizing we should add a comment above this line to explain why we're updating npm.
It is to get the version of npm that supports `npm ci`